mllib/src/main/scala/org/apache/spark/ml/feature/ChiSqSelector.scala

```diff
@@ -72,11 +72,15 @@ private[feature] trait ChiSqSelectorParams extends Params
   def getPercentile: Double = $(percentile)
 
   /**
-   * The highest p-value for features to be kept.
-   * Only applicable when selectorType = "fpr".
+   * alpha means the highest p-value for features to be kept when select type is "fpr".
```

Comment:
We should keep ```Only applicable when selectorType = "fpr", "fdr" or "fwe".```, since it's not applicable to other selector types such as "kbest" and "percentile".